tasties/urls.py

```diff
@@ -13,11 +13,11 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
 from django.urls import path
 from tasties_app import views
+from tasties_app.admin import Tasties_site
```

Comment:
Careful with naming... I think the name `Tasties_site` should be changed to `tasties_site` where it's defined. Please double check me against the naming conventions.

Reply:
I changed the name of the variable to what you suggested.